vmaas_sync/advisory_sync.go

```diff
@@ -60,14 +87,19 @@ func parseAdvisories(data map[string]vmaas.ErrataResponseErrataList) (models.Adv
 			utils.Log().Error("An advisory without description or summary")
 			continue
 		}
+		var severityID *int
+		if id, has := severities[strings.ToLower(v.Severity)]; has {
+			severityID = &id
+		}
 
 		advisory := models.AdvisoryMetadata{
 			Name:           n,
-			AdvisoryTypeID: advisoryTypes[v.Type],
+			AdvisoryTypeID: advisoryTypes[strings.ToLower(v.Type)],
```

Comment:
Why isn't it also checked as severityID?

Reply:
It's always non-null & there is foreign key. If there is invalid input, it sets advisory type to 0, and fails to save the metadata.